docs/release-notes/multirm-for-k8s.rst

```diff
@@ -0,0 +1,19 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes: Add ability to set up Determined master service on one Kubernetes cluster and
+   schedule workloads across the same or other Kubernetes clusters. Additional non-default resource
+   managers and resource pools are configured under the master configuration options
+   ``additional_resource_managers`` and ``resource_pools`` (additional resource managers are
+   required to have at least one resource pool defined). Additional resource managers and their
+   resource pools must have unique names.
```

Comment:
```suggestion
   ``additional_resource_managers`` and ``resource_pools``. (Add link to docs)
```

I think its fine to keep it high level and link to the docs

Reply:
@tara-det-ai are we linking to docs in release notes?

Reply:
let's do both. humans scan release notes. searchers like headings. a little repetition can't hurt.

Add this to release notes: 

For more information, visit :ref:`master configuration <master-config-reference>`.